ee/server/services2/ldap/service.ts

```diff
@@ -0,0 +1,16 @@
+import '../../broker';
+
+import { api } from '../../../../server/sdk/api';
```

Comment:
the `ee/server/services/*` doesn't work when being imported is bacause of:

https://github.com/RocketChat/Rocket.Chat/blob/0a293e547fcab88d548d14b6c8608b530b5d6f9d/.meteorignore#L1


the idea is that all EE services were intended to run on a separate process, which is not the case anymore with thi LDAP EE service..

with that in mind, I think we can use the name `local-services` or `internal-services` .. what do you think?